RoadMap_Movie/RoadMap_Movie/Cells/MovieDetailTableViewCell.swift

```diff
@@ -99,12 +99,16 @@ final class MovieDetailTableViewCell: UITableViewCell {
         return text
     }()
 
-    private let imdbButton: UIButton = {
+    private let networkService = MovieNetworkService()
+
+    private lazy var imdbButton: UIButton = {
         let button = UIButton(type: .custom)
         button.setImage(UIImage(named: Constants.imdb), for: .normal)
         return button
     }()
 
+    var presenter: DetailViewPresenterProtocol?
```

Comment:
```suggestion
    var presenter: DetailViewPresenterProtocol?
```
public распологается выше private и не забыть про марки)